packages/eas-cli/src/credentials/ios/actions/SetUpAdhocProvisioningProfile.ts

```diff
@@ -109,11 +111,17 @@ export class SetUpAdhocProvisioningProfile {
     );
 
     // 4. Reuse or create the profile on Apple Developer Portal
+    const applePlatform = await getApplePlatformFromSdkRoot(ctx.target);
+    const profileType =
+      applePlatform === ApplePlatform.TV_OS
+        ? ProfileType.TVOS_APP_ADHOC
+        : ProfileType.IOS_APP_ADHOC;
```

Comment:
So this is probably a good example of why `getApplePlatformFromSdkRoot` should return `IOS` by default (and not null).